mscore/pianotools.cpp

```diff
@@ -437,5 +442,41 @@ bool HPiano::gestureEvent(QGestureEvent *event)
             }
       return true;
       }
+
+//---------------------------------------------------------
+//   changeSelection
+//---------------------------------------------------------
+
+void PianoTools::changeSelection(const QList<Element *> &elements)
+      {
+      QSet<int> pitches;
+      switch(elements.size())
+            {
+            case 0:     break;
+
+            case 1:     if (elements.front()->type() != Ms::Element::Type::NOTE) break;
+                        for (auto n : static_cast<Note*>(elements.front())->chord()->notes())
+                              pitches.insert(n == elements.front() ? n->pitch() : n->pitch() << 8);
+                        break;
+
+            default:    {
+                        if (elements.front()->type() != Ms::Element::Type::NOTE) break;
+                        QSet<Element*> selected;
+                        Chord* chord = static_cast<Note*>(elements.front())->chord();
+                        for (auto e : elements)
+                              {
+                              if (e->type() != Element::Type::NOTE || static_cast<Note*>(e)->chord() != chord)
```

Comment:
in master we're using `isNote()` and `toNote()` instead, here and in other places of this PR

Reply:
There is no isNote() and toNote() methods at MuseScore 2.2

Reply:
Ah, didn't notice this was o(only) for 2.2